tests/libvmifact/factory.go

```diff
@@ -76,7 +76,14 @@ func NewAlpine(opts ...libvmi.Option) *kvirtv1.VirtualMachineInstance {
 		libvmi.WithRng(),
 	}
 	alpineOpts = append(alpineOpts, opts...)
-	return libvmi.New(alpineOpts...)
+	vmi := libvmi.New(alpineOpts...)
+
+	// Supplied with no user data, AlpimeWithTestTooling image takes more than 200s to allow login
+	if libvmi.GetCloudInitVolume(vmi) == nil {
+		withNonEmptyUserData := libvmi.WithCloudInitNoCloud(WithDummyCloudForFastBoot())
+		withNonEmptyUserData(vmi)
+	}
+	return vmi
```

Comment:
Can you please split changes not related to s390x into a separate commit

Reply:
done